test/fixtures/uphold_connections.yml

```diff
@@ -1,8 +1,22 @@
+<% salt = SecureRandom.random_bytes(12) %>
+
 unconnected:
   created_at: 2019-01-01
   publisher: completed
-  is_member: true
-  address: "abcdabcd-0000-0000-0000-abcd00000001"
+  is_member: false
+
+suspended_connection:
```

Comment:
You can simplify the file if you use default args like `user_authentication_tokens.yml`

Reply:
Going to push this off till later because it doesn't hurt anything to keep this mock data like this